website/static/js/nodesPrivacy.js

```diff
@@ -0,0 +1,272 @@
+/**
+ * Controller for changing privacy settings for a node and its children.
+ */
+'use strict';
+
+var $ = require('jquery');
+var ko = require('knockout');
+var bootbox = require('bootbox');
+var Raven = require('raven-js');
+
+var oop = require('./oop');
+var $osf = require('./osfHelpers');
+var Paginator = require('./paginator');
+var osfHelpers = require('js/osfHelpers');
+var m = require('mithril');
+var Treebeard = require('treebeard');
+var NodesPrivacyTreebeard = require('js/nodesPrivacySettingsTreebeard');
+
+//Todo (billyhunt): change url to dev or production as needed
+var API_BASE = 'http://localhost:8000/v2/nodes/';
```

Comment:
This needs to be changed before this is merged. See how other JS modules communicate with APIv2